### ISSUE TITLE
[py26] pip not installed if --system-site-packages is specified

### ISSUE THREAD

**Issue by bhuvan** [pypa/virtualenv#550]
This is with virtualenv-1.11. virtualenv don't install pip command when --system-site-packages is specified. The requirements are installed using system pip in system directories and eventually defeating the purpose of virtualenv. 

This happen when pip is installed within python.

bhuvan@build ~]$ /usr/bin/python2.6
Python 2.6.6 (r266:84292, Sep 11 2012, 05:13:24) 
[GCC 4.4.6 20120305 (Red Hat 4.4.6-4)] on linux2
Type "help", "copyright", "credits" or "license" for more information.

> > > import pip
> > > pip.**version**
> > > '1.4.1'

In this case, the pip command is not installed in venv.

INSTALL LOG with venv-1.11 and py26.
[bhuvan@build ~]$ virtualenv --version
1.11
[bhuvan@build ~]$ virtualenv --system-site-packages --python /usr/bin/python2.6 py26-venv-1.11 && ls py26-venv-1.11/bin
Running virtualenv with interpreter /usr/bin/python2.6
New python executable in py26-venv-1.11/bin/python2.6
Also creating executable in py26-venv-1.11/bin/python
Installing setuptools, pip...done.
activate  activate.csh  activate.fish  activate_this.py  python  python2  python2.6

INSTALL LOG with venv-1.11 and py27.
bhuvan@build ~]$ virtualenv --system-site-packages --python /usr/local/csi/bin/python2.7 py27-venv-1.11 && ls py27-venv-1.11/bin
Running virtualenv with interpreter /usr/local/csi/bin/python2.7
New python executable in py27-venv-1.11/bin/python2.7
Also creating executable in py27-venv-1.11/bin/python
Installing setuptools, pip...done.
activate  activate.csh  activate.fish  activate_this.py  pip  pip2  pip2.7  python  python2  python2.7

It looks more like a regression caused by 01fed47f5599f7aad777dbc709d7b30358628da7 when wheels support was added. The code should be fixed to install pip command explicitly into venv, even though it was installed as part of python.

**Comment by arthur-e** [comment]
I had this same issue on Ubuntu 14.04 with virtualenv 1.11 (installed a long time ago system-wide with `sudo pip install` outside the virtual environment), pip 6.07, and Python 3.4. When I tried `sudo pip uninstall virtualenv` I was told it wasn't installed, even though `which virtualenv` told a different story.

I did a `sudo find / -name virtualenv` and found that the binary was the ONLY thing installed so I just removed it, upgraded pip (as pip suggested) and then used pip to install virtualenv again. This fixed the problem. I'm guessing that my virtualenv got corrupted somewhere along my journey.
